phpstan.neon

```diff
@@ -0,0 +1,38 @@
+includes:
+	- phpstan-baseline.neon
+
+parameters:
+    level: max
+
+    checkFunctionNameCase: true
+    checkInternalClassCaseSensitivity: true
+    treatPhpDocTypesAsCertain: false
+
+    paths:
+        - application
+        - library
+
+    scanDirectories:
+        - vendor
+
+    ignoreErrors:
+        -
+            messages:
+                - '#Unsafe usage of new static\(\)#'
+                - '#. but return statement is missing#'
+            reportUnmatched: false
+
+        - '#Call to an undefined method React\\Promise#'
+
+        - '#Call to an undefined method Icinga\\Application\\ApplicationBootstrap::getRequest\(\)#'
+
+        - '#Call to an undefined method ipl\\Sql\\Connection::lastInsertId\(\)#'
+
+        - '#Access to an undefined property React\\Socket\\ConnectionInterface::\$stream#'
```

Comment:
These are vendor errors, please add these to baseline.

Reply:
I don't get it? Why would you add these to the baseline?

Reply:
Because that's what we did in all other modules. I personally prefer to keep the `phpstan.neon` clean.  
In other modules, the `phpstan.neon` file contains only errors that we do not consider as errors (e.g. `..return statement is missing`). But if you think that there is a very high probability that these errors will occur again and again, then it's fine, but I don't think so.

Reply:
As discussed offline, please fix error message in line 27.